components/script/dom/mutationrecord.rs

```diff
@@ -21,7 +21,7 @@ pub struct MutationRecord {
 }
 
 impl MutationRecord {
-    fn new(window: &Window, record_type: DOMString, target: Root<Node>) -> MutationRecord {
+    fn new(window: &Window, record_type: DOMString, target: JS<Node>) -> MutationRecord {
```

Comment:
The `target` argument should be `&Node` instead. You can use `JS::from_ref(target)` for the member initializer.